provider/aplus.py

```diff
@@ -105,7 +108,7 @@ def request_template_content(url):
 # "|en:<external url to template in english>|fi:<external url to template in finnish>|"
 # Therefore, if i18n exercise template urls are found,
 # we use a heuristic to first select the english template and if not available, the finnish template, and if not available, fail miserably
-def load_exercise_template_from_api_data(exercise_data):
+def load_exercise_template_from_api_data(exercise_data, course):
```

Comment:
Below, in the function `load_exercise_template()`, this function `load_exercise_template_from_api_data()` is called, but you have not updated the parameters in the call. Doesn't that cause a crash in `load_exercise_template()`? Please check this.

I guess `load_exercise_template()` already has easy access to the `course` object via `exercise.course` there.